packages/scripts/src/commands/app-data/copy.ts

```diff
@@ -25,16 +25,11 @@ const ASSETS_GLOBAL_FOLDER_NAME = "global";
  * CLI
  * @example yarn
  *************************************************************************************/
-interface IProgramOptions {
-  skipAssets: boolean;
-}
+interface IProgramOptions {}
 const program = new Command("copy");
-export default program
```

Comment:
This file contains the most refactoring, as the _copy_ step previously covered a host of different things including copying translation files, applying the translations, quality-checking assets and tidying code. Still a bit more work is required in the future, but in a better shape for now